internal/cli/tenants.go

```diff
@@ -14,9 +16,41 @@ func tenantsCmd(cli *cli) *cobra.Command {
 	}
 
 	cmd.AddCommand(useTenantCmd(cli))
+	cmd.AddCommand(listTenantCmd(cli))
 	return cmd
 }
 
+func listTenantCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "list",
```

Comment:
I'd suggest adding the `ls` command alias.

Reply:
Good idea, we could alias `ls` all the `list` commands.

Reply:
do you want to create a seperate ticket for that? `ls` all the `list` commands. I will make the changes for this PR

Reply:
Yup, I'll create a ticket for that